model_builder/model_builder/tests/unit_tests/test_model_builder_base.py

```diff
@@ -790,75 +790,75 @@ def test_make_parent_dirs__one_missing_dir(self):
+        test_base = GillesPy3DBase(path=self.test_folderpath)
+        with mock.patch('model_builder.handlers.util.GillesPy3DBase.get_dir_name') as mock_get_dir_name:
             mock_get_dir_name.return_value = test_path
             test_base.make_parent_dirs()
             self.assertIn("test_folder1", os.listdir(self.test_folderpath))
             self.assertIn("test_folder2", os.listdir(os.path.join(self.test_folderpath, "test_folder1")))
 
     ################################################################################################
-    # Unit tests for the StochSS base class print_logs function.
+    # Unit tests for the GillesPy3D base class print_logs function.
     ################################################################################################
 
     def test_print_logs__debug(self):
         ''' Check if debug log is logged correctly. '''
-        log = logging.getLogger('stochss')
-        test_base = StochSSBase(path=self.test_folderpath)
+        log = logging.getLogger('model_builder')
+        test_base = GillesPy3DBase(path=self.test_folderpath)
         test_base.log("debug", "testing debug log")
-        with mock.patch("stochss.handlers.log.log.debug") as mock_debug_log:
+        with mock.patch("model_builder.handlers.log.log.debug") as mock_debug_log:
             test_base.print_logs(log)
             mock_debug_log.assert_called_once_with("testing debug log")
 
 
     def test_print_logs__info(self):
         ''' Check if info log is logged correctly. '''
-        log = logging.getLogger('stochss')
-        test_base = StochSSBase(path=self.test_folderpath)
+        log = logging.getLogger('model_builder')
+        test_base = GillesPy3DBase(path=self.test_folderpath)
         test_base.log("info", "testing info log")
-        with mock.patch("stochss.handlers.log.log.info") as mock_debug_log:
+        with mock.patch("model_builder.handlers.log.log.info") as mock_debug_log:
             test_base.print_logs(log)
             mock_debug_log.assert_called_once_with("testing info log")
 
 
     def test_print_logs__warning(self):
         ''' Check if warning log is logged correctly. '''
-        log = logging.getLogger('stochss')
-        test_base = StochSSBase(path=self.test_folderpath)
+        log = logging.getLogger('model_builder')
+        test_base = GillesPy3DBase(path=self.test_folderpath)
         test_base.log("warning", "testing warning log")
-        with mock.patch("stochss.handlers.log.log.warning") as mock_debug_log:
+        with mock.patch("model_builder.handlers.log.log.warning") as mock_debug_log:
             test_base.print_logs(log)
             mock_debug_log.assert_called_once_with("testing warning log")
 
 
     def test_print_logs__error(self):
         ''' Check if error log is logged correctly. '''
-        log = logging.getLogger('stochss')
-        test_base = StochSSBase(path=self.test_folderpath)
+        log = logging.getLogger('model_builder')
```

Comment:
Change `log = logging.getLogger('model_builder')` to `log = logging.getLogger('gillespy3d')`